test/hybig/HyBIG_Regression.ipynb

```diff
@@ -263,9 +277,7 @@
     "\n",
     "        expected_output_metadata = {\n",
     "            'driver': 'JPEG', **common_aster_metadata, 'count': 3\n",
-    "        }\n",
-    "        print(expected_output_metadata)\n",
-    "        \n",
+    "        }        \n",
     "        with rasterio.open(test_files[0]) as test_file:\n",
```

Comment:
Totally not from your PR, but just noticed this extraneous print statement in this test.